.devcontainer/devcontainer.json

```diff
@@ -21,14 +21,19 @@
 		"mhutchie.git-graph",
 		"tabnine.tabnine-vscode",
 		"redhat.java",
-		"redhat.vscode-commons"
+		"redhat.vscode-commons",
+		"ms-vscode.cpptools",
+		"ms-vscode.cmake-tools"
 	],
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
 	// "forwardPorts": [8080, 9090, 46753, 46800, 5555, 6666, 8090, 2181, 9092],
 	// Use 'postCreateCommand' to run commands after the container is created.
 	"postCreateCommand": "bash .devcontainer/post-create.sh",
 	// Comment out connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
 	// "remoteUser": "vscode"
+	"mounts": [
+		"source=C:\\ppm_data,target=/ppm_data,type=bind,consistency=cached"
```

Comment:
Is this assuming you have the ppm_data directory at the root C:\ level?

Reply:
Yes

Reply:
Is that mount needed for the ppm module to function? Can we pull it in from a relative path instead of hard coding it?

Reply:
This mount is necessary only when running the PPM through its dev container or the ODE's dev container. We might be able to pull it from a relative path, I can look into that.

Reply:
It turns out that relative paths cannot be used for directory mounts in devcontainer.json.

Reply:
This mount has been removed.